src/clojure_tic_tac_toe/core.clj

```diff
@@ -1,16 +1,31 @@
 (ns clojure-tic-tac-toe.core
-  (:gen-class))
+  (:require [clojure-tic-tac-toe.input_output :as io]
+            [clojure-tic-tac-toe.board :as board]))
 
-(defn board-to-string []
-  (clojure.string/join "\n" [" 1 | 2 | 3 "
-                             "---+---+---"
-                             " 4 | 5 | 6 "
-                             "---+---+---"
-                             " 7 | 8 | 9 "]))
+(defn get-move
+  [player]
+  (io/prompt-player-for-move player)
+  (flush)
+  (io/get-move))
+
+(defn int-to-keyword
```

Comment:
This probably belongs in `io` as part of `get-move`.   The way it is now, the responsibility of getting a move from stdin is split across multiple files.

Reply:
Moved functionality for `core.clj` `get-move` and `int-to-keyword` to `get-move` in `input_output.clj`